### DISCUSSION TITLE
y21ga_0.2.2 update failed

### DISCUSSION THREAD

**Discussion by tmp-tmp-tmp** [roleoroleo/yi-hack-Allwinner-v2#443] [General]
After updating to y21ga_0.2.2, all my cameras began to work for 3 minutes and reboot! Had to go back to 0,2,1
Base Version 9.0.19.02_202007141512
Model Suffix y21ga
Serial Number IFUSY44

Except for one old
Base Version 9.0.19.02_202007141512
Model Suffix y21ga
Serial Number BFUSY44

What's going on?

I noticed that before the reboot, free RAM starts to run out quickly.

**Comment by roleoroleo** [comment]
There are no big changes between 0.2.1 and 0.2.2.
If you update the cam with ota update, try to install the hack from scratch.
Did you enable the swap file?

**Reply by tmp-tmp-tmp** [reply]
I tried OTA and SD update. Both methods lead to the same result.
SWAP is off. I'll try and let you know. The camera is located high and there is no way to turn it off for a long time.

**Comment by gl2007** [comment]
Would enabling swap file do this behavior? I did enable swap file yesterday
in my new Yi 1080p Home BFUS camera but have not seen it yet but want to be
cautious.

**Comment by roleoroleo** [comment]
There are some features that use a lot of ram and, if you don't enable swap file, the cam runs out of memory.
For example: snapshot and thumbnails.
